src/scripts/imageScaling.js

```diff
@@ -0,0 +1,21 @@
+function resizedataURL(datas, wantedWidth, wantedHeight){
+    return new Promise(async function(resolve,reject){
+
+        var img = document.createElement('img');
+
+        img.onload = function()
+        {        
+            var canvas = document.createElement('canvas');
+            var ctx = canvas.getContext('2d');
+            canvas.width = wantedWidth;
+            canvas.height = wantedHeight;
+            ctx.drawImage(this, 0, 0, wantedWidth, wantedHeight);
+
+            var dataURI = canvas.toDataURL();
+
+            resolve(dataURI);
+        };
+        img.src = datas;
+
+    })
+}// 
```

Comment:
Is there meant to be a // ?

Reply:
Nope, this has been corrected and committed.